service/controller/v15/resource/deployment/master_deployment.go

```diff
@@ -165,13 +165,12 @@ func newMasterDeployments(customObject v1alpha1.KVMConfig) ([]*extensionsv1.Depl
 								Image:           key.K8SEndpointUpdaterDocker,
 								ImagePullPolicy: apiv1.PullIfNotPresent,
 								Command: []string{
-									"/bin/sh",
-									"-c",
-									"/opt/k8s-endpoint-updater update --provider.bridge.name=" + key.NetworkBridgeName(customObject) +
```

Comment:
you could also just run `exec /opt/k8s-endpoint-updater`. then you wouldn't have to change the flag in the endpoint-updater.

Reply:
would exec also interpret the env value?

Reply:
the env value should be fine. if signal handling works fine :man_shrugging:. was just an idea.

Reply:
When overriding `ENTRYPOINT` and `CMD` (that are defined in docker container), isn't it then executing given command as such and then it must be full path to executable in container. There can't be any shell in between because that would be implicit requirement for containers and I'm not aware of anything like this. Or am I missing something?

Reply:
Yes it is like that, so  looks like exec is not usable